### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by mjoyce91**
Increases coverage to 70% by adding new tests and removing deprecated code (XML import functions).

Screen Shot 2020-11-16 at 2 35 18 PM

### CODE REVIEW

File: talentmap_api/feature_flags/tests/test_featureflags.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import pytest
+from model_mommy import mommy
+from rest_framework import status
+
+from talentmap_api.feature_flags.models import FeatureFlags
+
+
+@pytest.fixture
+def test_featureflags_fixture():
+    if FeatureFlags.objects.first() is None:
```
Comment (elizabeth-jimenez) [inline talentmap_api/feature_flags/tests/test_featureflags.py:11]: so these tests actually pull from the DB? very cool

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🥮 🍬 🍥  👍

**Comment by elizabeth-jimenez** [comment]
Very informative 😄 

Just a ?, did we remove all that stuff from `talentmap_api/common/xml_helpers.py` cause it's no longer relevant?

**Comment by mjoyce91** [comment]
@elizabeth-jimenez 
>Just a ?, did we remove all that stuff from talentmap_api/common/xml_helpers.py cause it's no longer relevant?

yes, way back when, we did XML imports of the data

**Pull request merged**